Doppler.ContactPolicies.Api/Controllers/ContactPolicies.cs

```diff
@@ -29,6 +29,12 @@ public async Task<IActionResult> GetContactPoliciesSettings(string accountName)
             if (idUser == null)
                 return NotFound($"Account {accountName} does not exist.");
 
+            var userHasContactPoliciesFeature = await _userFeaturesService.HasContactPoliciesFeatureAsync(accountName);
+            if (!userHasContactPoliciesFeature)
+            {
+                return new ObjectResult($"User with Account {accountName} has not contact policies feature.") { StatusCode = 404 };
```

Comment:
Could you please write it in two lines to make the status code more evident?

Also, in my opinion, it is a 403 error because the user does not have permission.

```suggestion
                return new ObjectResult($"User with Account {accountName} has not contact policies feature.")
                {
                    StatusCode = 404
                };
```

Reply:
Done! Thanks!

Reply:
I included a new test to check this situation: https://github.com/FromDoppler/doppler-contact-policies-api/pull/60/commits/dc1da4870ef72fc9486011f2fdd4a574426214d3